### ISSUE TITLE
About the parameter setting regarding loss functions

### ISSUE THREAD

**Issue by LPengYang** [xiaofeng94/RefineDNet-for-dehazing#5]
Thank you for your excellent job with the single image dehazing task. and I am a little confused about the parameter setting in the model. In the published paper,  three loss functions were adopted,  and the weight values are 0.02 for GAN loss、1 for Reconstruction loss、and 1 for Identity loss, respectively（Eq.19 in the paper).  However, in the provided code (model/refined_DCP_model.py), four loss functions are adopted, and the weights are 0.05 for GAN loss、1 for Reconstruction loss、 1 for Identity loss 、1 for TV  loss, respectively.  We are confused about that which parameters setting is suggested to reproduce the provided pre-trained model. Additionally, we notice that the batch size, which is not mentioned in the paper, is set as 1 in the provided code,  if such setting is also suggested for a good reproduction. Thank you again for your significant contribution to single image dehazing.

**Comment by xiaofeng94** [comment]
Hey,

Both 0.05 and 0.02 are fine. Generally, we found 0.05 brings about better results. For this hyperparameter, you may find some discussion in Fig.9(a) of our paper.

Like CyclGAN, the default batch size is 1. We did not try other batch sizes.

**Comment by LPengYang** [comment]
Thank you so much for your detailed explanation. In our experiments, it is found that, with the provided code,  0.05 is fine for a good reproduction, which further verifies your excellent contribution. However, 0.02 appears to show a huge drop, if such a situation happened before or something appears to contribute to this error. Additionally, similar to CycleGAN, the work does not use any paired images and is trained in an unsupervised way, so we are confused that why it is classified to the weakly supervised methods and if it is available to classify this work (as well as the DisentGAN) to unsupervised methods.
Thank you again for your meaningful contributions.

**Comment by xiaofeng94** [comment]
Hey,

We call this method weakly supervised because it requires humans to make sure if an image is hazy or haze-free. That's kind of image-level annotations, while previous learning-based methods require paired images that provide pixel-level supervision.

**Comment by LPengYang** [comment]
Thank you very much for your answers.
